### PR TITLE
Can click receipt on main page to get to overview page

### PR DESCRIPTION
**Pull request by ShriGrover**

### CODE REVIEW

File: app/src/main/java/com/billbreaker/EditItemsActivity.java [hunk 1]
```diff
@@ -197,11 +165,11 @@ public void onSwiped(RecyclerView.ViewHolder viewHolder, int direction, final in
...
-            if (direction == RIGHT) {
-                showEditItemsPopup(modifiedIndex, modifiedItem);
-//                mAdapter.removeItem(modifiedIndex + 1);
-                undoMessageText = " edited!";
-            }
+//            if (direction == RIGHT) {
+//                showEditItemsPopup(modifiedIndex, modifiedItem);
+////                mAdapter.removeItem(modifiedIndex + 1);
+//                undoMessageText = " edited!";
+//            }
```
Comment (adayyli) [inline app/src/main/java/com/billbreaker/EditItemsActivity.java]: Remove code if commented out

Comment (ShriGrover) [reply app/src/main/java/com/billbreaker/EditItemsActivity.java]: @annaowens should this be removed?

File: app/src/main/java/com/billbreaker/MainActivity.java [hunk 2]
```diff
@@ -63,7 +72,18 @@ public void onClick(View view) {
...
-        // receipts = receiptDatabase.getAllReceipts(); TODO uncomment this when ada updates receipt database
+         receipts = receiptDatabase.getAllReceipts();
+//        PersonalReceiptItem shri = new PersonalReceiptItem("shri", 13.34);
+//        PersonalReceiptItem ada = new PersonalReceiptItem("ada", 14.57);
+//        PersonalReceiptItem amrit = new PersonalReceiptItem("amrit", 9.83);
+//        PersonalReceiptItem anna = new PersonalReceiptItem("anna", 11.95);
+//        List<PersonalReceiptItem> items = Arrays.asList(shri, ada, amrit, anna);
+//        Receipt receipt = new Receipt(items, Calendar.getInstance(TimeZone.getTimeZone("UTC")).getTimeInMillis(), 0, 0, 0);
+//
+//        receipts = Arrays.asList(receipt, receipt, receipt);
```
Comment (adayyli) [inline app/src/main/java/com/billbreaker/MainActivity.java:82]: same comment as above

File: app/src/main/java/com/billbreaker/EditItemsActivity.java (hunk 1 above)
Comment (adayyli) [reply app/src/main/java/com/billbreaker/EditItemsActivity.java]: she said no. please delete

File: app/src/main/java/com/billbreaker/MainActivity.java (hunk 2 above)
Comment (ShriGrover) [reply app/src/main/java/com/billbreaker/MainActivity.java:82]: keeping this in for more testing in the future

**Pull request merged**